chore: deprecate embedded plugin registry (#3019)
codenvy/che-docs: modules/administration-guide/partials/proc_adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance.adoc

[IMPORTANT]
====

> You can manage Visual Studio Code extensions by setting up and using an internal, on-premises Open VSX registry. This approach provides full control over the extension lifecycle, enables offline use, and improves compliance. The embedded plugin registry is deprecated, with the Open VSX registry serving as its successor. Refer to the xref:running-the-open-vsx-on-premises.adoc[] for detailed setup instructions.

Reviewer: ```suggestion
You can manage Visual Studio Code extensions by setting up an internal, on-premises Open VSX Registry. This approach provides full control over the extension lifecycle, enables offline use, and improves compliance. The embedded plugin registry is deprecated; the Open VSX Registry is its successor. See xref:running-the-open-vsx-on-premises.adoc[Running the Open VSX Registry on premises] for detailed setup instructions.
```
